tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/NLog/DirectSubmission/Proxies/IJson5LayoutProxy.cs

```diff
@@ -0,0 +1,21 @@
+// <copyright file="IJson5LayoutProxy.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Logging.NLog.DirectSubmission.Proxies
+{
+    internal interface IJson5LayoutProxy : IDuckType
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether to include the contents of the ScopeContext properties-dictionary.
+        /// </summary>
+        /// <docgen category='Layout Options' order='10' />
+        bool IncludeScopeProperties { get; set; }
+
+        [DuckField(Name = "_includeScopeProperties")]
+        bool IncludeScopePropertiesField { get; } // added in v5
```

Comment:
Ah, I think I understand 🤔 I'm wondering if you can do the following and achieve the same thing? Depends if they're simple properties or not, I'll defer to you, it's not important, was only in case it was an accident you had both 🙂 

```csharp
        [DuckField(Name = "_includeScopeProperties")]
        bool IncludeScopePropertiesField { get; set; } // include the setter here?
```

Reply:
Yep that worked